ci/validate-pr-title/validate.js

```diff
@@ -8,7 +8,6 @@ const allowedTypes = [
   "perf",
   "test",
   "ci",
-  "chore",
```

Comment:
Why are we removing chore?

Reply:
merely removing a duplicate, `chore` is already added on line 4.

found it, thought to clean